### PR TITLE
Adding in support for VNC to view running Behat tests.

### PR DESCRIPTION
**Pull request by rlorenzo**
We had some of our talented student developers tackle on enabling someone to view Behat tests using VNC. Check it out.

### CODE REVIEW

File: base.yml [hunk 1]
```diff
@@ -20,6 +20,9 @@ services:
       POSTGRES_PASSWORD: "m@0dl3ing"
       POSTGRES_DB: moodle
   selenium:
-    image: selenium/standalone-firefox:2.53.1
+    image: selenium/standalone-firefox-debug:2.53.1
+    build: ./selenium/
```
Comment (danpoltawski) [inline base.yml]: I think that image and build are conflicting here? If this is working I assume that build is winning as the last one.

**Comment by danpoltawski** [comment]
Hi Rex/Student Developers,

1) My gut feel is that we don't want this by default, but should control it with some debug option. This might get complex.
2) I don't quite understand why you are doing the build and setting the vnc password to test - https://github.com/SeleniumHQ/docker-selenium#debugging suggests you can just use the debug images and the password is `secret`

**Comment by abias** [comment]
While I haven't tested this patch myself, I would really be interested in this feature for manual testing. 

> My  gut feel is that we don't want this by default, but should control it with some debug option. 

Or perhaps you could control that with another environment variable for /bin/moodle-docker-compose

**Comment by danpoltawski** [comment]
> While I haven't tested this patch myself, I would really be interested in this feature for manual testing.

Why not use your own browser for that? I don't feel like using selenium node is best solution for this.

> Or perhaps you could control that with another environment variable for /bin/moodle-docker-compose

Yes, sorry I wasn't clear - that is what I meant.

File: base.yml (hunk 1 above)
Comment (rlorenzo) [reply base.yml]: So we just need build and not image since the Dockerfile in build is already there?

**Comment by rlorenzo** [comment]
@danpoltawski Okay, we didn't know that there was a set password.

But running a Behat test can be much different than running a manual test and there are times in which you want to see the test run to understand why it is failing.

Also, we cannot run the "And I pause" step using Docker. Any ideas on how to enable it?

**Comment by abias** [comment]
> Why not use your own browser for that? I don't feel like using selenium node is best solution for this.

Let me rephrase that in accordance with Rex' use case:
I would really be interested in this feature for seeing what the Behat tests are doing visually and not only reading in the terminal that tests have passed.

File: base.yml (hunk 1 above)
Comment (danpoltawski) [reply base.yml]: Yep, the image is not being used, you either use an image or build.

But for the reasons I mention below, I don’t think you need to build anyway.

**Comment by danpoltawski** [comment]
I've merged a slightly changed version of this in #49 - thanks a lot for the patch Rex

https://github.com/moodlehq/moodle-docker#using-vnc-to-view-behat-tests
